mithril-aggregator/src/dependency_injection/builder.rs

```diff
@@ -1312,6 +1317,16 @@ impl DependenciesBuilder {
         Ok(self.single_signer_authenticator.as_ref().cloned().unwrap())
     }
 
+    fn get_epoch_settings_configuration(&mut self) -> StdResult<AggregatorEpochSettings> {
```

Comment:
A specialized result for dependency building is available :) (and is a dropping replacement here since it automatically convert from StdResult).
```suggestion
    fn get_epoch_settings_configuration(&mut self) -> Result<AggregatorEpochSettings> {
```

Reply:
Done ✅